src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+
+    private static final FigureSupplier figureSupplies = new FigureSupplier();
+
+    public static void main(String[] args) {
+
+        Figure getRandomFIgure = figureSupplies.getRandomFigure();
```

Comment:
The variable name `getRandomFIgure` does not follow Java naming conventions. Variable names should not start with a verb and should be in camelCase. Consider renaming it to `randomFigure`.